.travis.yml

```diff
@@ -14,7 +14,7 @@ script:
     - julia -e 'Pkg.add("Images")'  # Needed for Docs 
     - julia -e 'Pkg.clone("https://github.com/JuliaImages/ImageDraw.jl")' # Needed for Docs 
     - julia -e 'Pkg.clone(pwd()); Pkg.build("ImageFeatures")'
-    - julia -e 'Pkg.test("ImageFeatures", coverage=true)'
+    - julia -e 'Pkg.test("ImageFeatures",coverage=true)'
```

Comment:
Beacuse the tests for orb.jl were taking too much time on travis, I used the `travis_wait` command. But the space present here caused problems with travis, so I had to remove it.